tools/topology/topology2/platform/intel/test-wov-config.conf

```diff
@@ -0,0 +1,25 @@
+# The sof_abi_hdr for ipc4 is:
```

Comment:
can you please remove this file and add the data block directly into wov.conf like we're doing for the smart amp?